templates/new/web_main.py.txt

```diff
@@ -17,8 +17,8 @@ assemblies = [
         .set_estimator(Baseline())
 ]
 
-def main():
-    config = Config(auto_load=True)
+@has_config
+def main(config):
```

Comment:
Set default value of config to None, main is called with no arguments at bottom of file.

Reply:
Done 👍